iothub/device/src/Common/Exceptions/DeviceMessageLockLostException.cs

```diff
@@ -8,10 +8,16 @@
 namespace Microsoft.Azure.Devices.Client.Exceptions
 {
     /// <summary>
-    /// The exception that is thrown when an attempt to communicate with a device fails because the lock token was lost (if the connection is lost and regained for example). This timeout has the same effect as if the message was abandonned.
+    /// The exception that is thrown when an attempt to communicate with a device fails because the lock token was lost
```

Comment:
nit: I don't particularly like the wording "...an attempt to communicate with a device fails..." that we had previously. The communication with the device is fine, it is rather telling you over that communication what the operation that you wanted to execute failed. 
"Communication with the device failed" reads to me as the communication act itself failed.

Reply:
Sure we can change the wording. What do you have in mind? How about just removing the first sentence altogether? We can start with "This exception is thrown when attempting to reject/abandon/complete ...." ?

Reply:
Let me know if you have other thoughts. I've resolved it by removing the first line.